.rubocop.yml

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
As mentioned, this is not our up to date config. You do not have access to that though, so this was pretty much an impossible task